compose-network/mirror-node/application.yml

```diff
@@ -17,3 +17,40 @@ hedera:
         endpointOverride: http://minio:9000
       initialAddressBook: "/usr/etc/hedera-mirror-importer/local-dev-1-node.addressbook.f102.json.bin"
       network: OTHER
+    monitor:
+      mirrorNode:
+        grpc:
+          host: mirror-node-grpc
+          port: 5600
+        rest:
+          host: mirror-node-rest
+          port: 5551
+      publish:
+        scenarios:
+          pinger:
+            properties:
+              amount: 1
+              maxTransactionFee: 10000
+              senderAccountId: 0.0.2
+              recipientAccountId: 0.0.55
+              transferTypes:
+                - CRYPTO
+            receiptPercent: 1.0
+            tps: 0.1
```

Comment:
I would suggest increasing this from every 10s to 1s to close the record file more often. Can use an env variable to customize it as well.

Reply:
2s might be better to better simulate current network behaviour

Reply:
The file closes every 2s but if you send a transaction every 2s it can sometimes take up to 4s to close then. Better to do 1s so it has a better chance to close at 2s more consistently.

Reply:
I have no issues with increasing the rate of submissions, however, I dont necessarily want to create excess noise either on the single node network. 

The noise will be fairly apparent once I add in the hashscan app dashboard.

Reply:
Well, the point of adding the monitor was ensure the record file would close fast enough so that users would not see a delay in viewing their transactions. If our goal is to minimize this latency as much as possible than 1s should be preferred. If delay is not a concern just that it does eventually close than every 10s might be fine. But I don't think 1 TPS is much noise and users can adjust to their needs.

Do we think we'll get more complaints of "I don't see my transaction show up until sometimes up until 20 seconds later" or "My API/database/UI has too much unrelated data"?

Reply:
Done TPS increased to `1.0`